python/dgl/distributed/partition.py

```diff
@@ -190,7 +190,14 @@ def load_partition(part_config, part_id, load_feats=True):
     assert (
         "part_graph" in part_files
     ), "the partition does not contain graph structure."
-    graph = load_graphs(relative_to_config(part_files["part_graph"]))[0][0]
+    partition_path = relative_to_config(part_files["part_graph"])
+    print(
+        f"Start to load partition from {partition_path} which is "
+        f"{os.path.getsize(partition_path)} bytes. It may take non-trivial "
+        "time for large partition."
+    )
+    graph = load_graphs(partition_path)[0][0]
+    print("Finished loading partition...")
```

Comment:
I think we don't need `...` here as we are not waiting for anything.

Reply:
removed